### PR TITLE
Adding comments.

### PR DESCRIPTION
**Pull request by smn**
This work is partly going on in http://github.com/praekelt/molo.comments but the TuneMe implementation bits will be reflected here.

### CODE REVIEW

**Comment by smn** [comment]
Not ready for review yet.

**Comment by smn** [comment]
cursor_and_this_is_an_article_with_no_image_hero_image_uploaded

**Comment by bruskiza** [comment]
aaah yeah... ready now? if so :+1:

**Comment by smn** [comment]
Not not ready for review yet.

**Comment by smarzban** [comment]
:clap: nice work

File: tuneme/templates/core/article_page.html [hunk 1]
```diff
@@ -39,14 +48,31 @@
...
+{% recursetree comment_list %}
+<hr>
+<div class="comment {{self.get_parent_section.get_effective_extra_style_hints}} {% if node.user.is_staff %}staff{% endif %}">
+  <p class="by"><strong>{{node.user.profile.alias}}</strong> <span class="date">{{node.submit_date|date:"j F Y"}}</span></p>
+  <p>{{node.comment}}</p>
+  <p><a href="{% url 'comments-report' node.pk %}" class="report">Report +</a></p>
 </div>
+{# recursion! children of a given comment #}
+{% if not node.is_leaf_node %}
+  {{ children }}
```
Comment (smn) [inline tuneme/templates/core/article_page.html:74]: This is where the threaded comments are rendered, they're rendered using the same HTML as above, it's recursively used as a template.

**Comment by smn** [comment]
@saeedmb8 @bruskiza @miltontony ready for review.
Doesn't meet everything required in the scope yet but we're getting closer.
This builds on the work done in https://github.com/praekelt/molo.commenting

**Comment by smarzban** [comment]
:+1: :ok_hand:

**Comment by bruskiza** [comment]
:+1:

**Comment by smn** [comment]
Created #9 for anonymous commenting, will address that there.

**Pull request merged**